test/TestUtilities/Test.Utility/AssemblyReader.cs

```diff
@@ -1,13 +1,14 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#if !IS_DESKTOP
```

Comment:
If this class is commented out because `System.Reflection.Metadata` isn't found for .NET Framework, it's a nuget package that can be referenced.

Reply:
Thanks! As of now, we don't have code reference the `AssemblyReader `in full framework. And there is no issue after removing the preprocessor.